UnityEngine.AssetBundles/Editor/AssetBundleBrowser/AssetBundleModel/ABModelAssetInfo.cs

```diff
@@ -173,7 +173,25 @@ public IEnumerable<MessageSystem.Message> GetMessages()
                 }
                 message = message.Substring(0, message.Length - 2);//remove trailing comma.
                 messages.Add(new MessageSystem.Message(message, MessageType.Info));
-            }
+            }            
+
+            if (m_dependencies != null && m_dependencies.Count > 0)
+            {
+                var message = string.Empty;
+                foreach (var dependent in m_dependencies)
+                {
+                    if (dependent.bundleName != bundleName)
+                    {
+                        message += dependent.bundleName + " : " + dependent.displayName + "\n";
+                    }
+                }
+                if (string.IsNullOrEmpty(message) == false)
+                {
+                    message = message.Insert(0, displayName + "\n" + "Is dependent on other bundle's asset(s) or auto included asset(s): \n");
```

Comment:
message just needs to not have "or auto included asset(s)".  If you have a test case that proves otherwise let me know, but this shouldn't happened. If AssetA depends on AssetB, then AssetB will be auto-included in the same bundle as AssetA unless AssetB is _explicitly_ included in another bundle.  If AssetB is auto-included in another bundle, then it will be auto-included in both, leading to the duplicate asset warning already present.

Reply:
oh. hmm. i think i misunderstood your message. ok, scratch that comment, I'll take it as is, but may later split it into two separate messages depending on if it's an "auto" or not.